conan1.0.rst

```diff
@@ -40,21 +40,19 @@ Deprecations/removals
 ---------------------
 
 - scopes were completely removed in conan 0.30.X
-- ``self.conanfile_directory`` has been removed. Use ``self.source_folder``, ``self.build_folder``, etc. instead
-- ``self.cpp_info``, ``self.env_info`` and ``self.user_info`` scope has been reduced to only the ``package_info()`` method
-- ``gcc`` and ``ConfigureEnvironment`` were already removed in conan 0.30.1
-- ``werror`` doesn't exist anymore. Now it is the builtin behavior.
-- Command ``test_package`` has been removed. Use :command:`conan create` and :command:`conan test` instead.
-- ``CMake`` helper only allows now (from conan 0.29). the ``CMake(self)`` syntax
-- :command:`conan package_files` command was replaced in conan 0.28 by :command:`conan export-pkg` command.
+- ``self.conanfile_directory`` has been removed. Instead, use ``self.source_folder``, ``self.build_folder``, etc.
+- ``self.cpp_info``, ``self.env_info`` and ``self.user_info`` scope has been reduced to only include the ``package_info()`` method
```

Comment:
The real sense of the sentence is something like "``self.cpp_info``, ``self.env_info`` and ``self.user_info`` now exist only inside the ``package_info()`` method.``

Reply:
something like would be better:
```
``self.cpp_info``, ``self.env_info`` and ``self.user_info`` scope has been reduced to the ``package_info()`` method exclusively.
```